views/components/Certification.js

```diff
@@ -1,72 +1,303 @@
-              English at the university level. And it evaluates how well you
-              combine your reading, listening, speaking and writing skills to
-              perform academic tasks.
-            </p>
-          </li>
-          <li className="process__step process__step_deploy">
-            <div className="process__step-icon icon icon_prize"></div>
-          </li>
-          <li className="process__step-info process__step-info_deploy">
-            <h3>
-              <spana>3</spana>
-              &mdash;Second place
-            </h3>
-            <p>
-              Got second prize for the competitive algorithm contest of the
-              university
-            </p>
-          </li>
-          <li className="process__step process__step_deliver">
-            <div className="process__step-icon icon icon_prize"></div>
-          </li>
-          <li className="process__step-info process__step-info_deliver">
-            <h3>
-              <spana>4</spana>
-              &mdash;First place
-            </h3>
-            <p>
-              Highschool, got First prize for the competitive algorithm contest
-              of the city.
-            </p>
-          </li>
-        </ul>
+import React, { useState, Fragment } from 'react'
+import styled from 'styled-components/macro'
+import CertIcon from '~/views/components/common/CertIcon'
+import { CERTIFICATIONS } from '~/consts/pages'
+import { CERT_ACTIVE_ITEM } from '~/consts/certification'
+import PropTypes from 'prop-types'
+import Container from '~/views/components/common/Container'
+import Header from '~/views/components/common/Header'
+import { Translate } from 'react-localize-redux'
+
+const Certification = () => {
+  const [itemSelected, setSelectedItem] = useState(CERT_ACTIVE_ITEM[0])
+
+  return (
+    <Container id={CERTIFICATIONS}>
+      <div className="container">
+        <Header>
+          <h2>Certifications and Prizes</h2>
+        </Header>
+        <List className="certification">
+          <Research
+            itemSelected={itemSelected}
+            setSelectedItem={setSelectedItem}
+            title="CertResearch"
+            content="CertContentResearch"
+          />
+          <Develop
+            itemSelected={itemSelected}
+            setSelectedItem={setSelectedItem}
+            title="CertDevelop"
+            content="CertContentDevelop"
+          />
+          <Deploy
+            itemSelected={itemSelected}
+            setSelectedItem={setSelectedItem}
+            title="CertDeploy"
+            content="CertContentDeploy"
+          />
+          <Deliver
+            itemSelected={itemSelected}
+            setSelectedItem={setSelectedItem}
+            title="CertDeliver"
+            content="CertContentDeliver"
+          />
+        </List>
       </div>
-    </div>
-  </section>
-)
+    </Container>
+  )
+}
 
 export default Certification
+
+const Research = ({ itemSelected, setSelectedItem, title, content }) => {
```

Comment:
extract these components to other folder like views/components/certification/  too

Reply:
views/components/certification/Research

Reply:
Also consider how to rename the component, Reseach, Develop, Deploy,... doesn't make any sense in the context
We colud use Prize_1 Prize_2 Cert_1 Cert_2
However consider reuse Prize component for 2 Prizes and Cert component for 2 Certs if we go this way 💃